apps/server/src/controllers/integrationController.ts

```diff
@@ -267,6 +271,37 @@ export function dispatchFromAdapter(
       return { topic: 'timer', payload: timer };
     }
 
+    case 'set-time': {
+      const { eventId, property, value } = payload as ChangeOptions;
+      const amout = Number(value);
+      if (isNaN(amout)) {
+        throw new Error(`set-time "${value}" is not a number`);
+      }
+      const event = EventLoader.getEventWithId(eventId);
```

Comment:
Would we not like to fail faster?

Maybe add a method in the playback service to get current event and compare that?